packages/react-components/src/components/SendBox.tsx

```diff
@@ -218,7 +218,12 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   return (
     <Stack
       className={mergeStyles(
-        borderAndBoxShadowStyle(theme, isDarkThemed(theme) ? '#f1707b' : '#a80000', !!errorMessage, !!disabled),
+        borderAndBoxShadowStyle({
+          theme,
+          errorColor: isDarkThemed(theme) ? '#f1707b' : '#a80000',
```

Comment:
I am curious  it looks like errorColor we want to use fixed values and it seems to be consistent. Should we just make it somehow part of "theme" by default?

Also we are being very specific about the colors that are used. Is there a palette color we can align to instead?

Reply:
Yeah not a fan of these hard coded colors either, filed a bug: [Bug 2779691](https://skype.visualstudio.com/SPOOL/_workitems/edit/2779691): [Web] Can sendbox error coloring use a theme value?